### ISSUE TITLE
Tab Order 

### ISSUE THREAD

**Issue by carinaluise** [substrate-developer-hub/substrate-docs#1305]
### The tab order skips the following page elements:
*  Header nav search
*  Color mode
*  Header nav children with sub-menus (Opportunities, Resources, Connect)
*  Banner close icon

### Following page elements are selected with tabbing, but have no focus styling:
*  Header nav items (Technology, Developers, Vision, Ecosystem)
*  Sidebar nav open/close icon

### Other tab order issues:
*  Banner is selected first
*  When using sidebar nav, tabbing position is reset on click
*  Table of Contents nav is after article in tab order
*  Footer: Social media icons are before Privacy Policy and ToS in tab order
* Hamburger Menu: when inside hamburger menu items can not be tabbed through- perhaps a similar issue to the dialog where tab index is on content behind?

**Comment by jonnysmillie** [comment]
Most of the issues raised here will be addressed by this PR: https://github.com/substrate-developer-hub/substrate-docs/pull/1210

 Banner is selected first - this is correct.
 When using sidebar nav, tabbing position is reset on click - this is correct.
 
 There are some others that I am still looking into:
 
 Table of Contents nav is after article in tab order
 Footer: Social media icons are before Privacy Policy and ToS in tab order
 Hamburger Menu: when inside hamburger menu items can not be tabbed through- perhaps a similar issue to the dialog where tab index is on content behind?

**Comment by jonnysmillie** [comment]
Closing this as I think it has been mostly resolved. If any other specific issues arise please crate a new issue or reopen this one.
